### ISSUE TITLE
Video captioning for individual videos using precomputed weights not working

### ISSUE THREAD

**Issue by sojhal** [rohit-gupta/Video2Language#8]
IOError: [Errno 2] No such file or directory: '../language_model/vocabulary_10.p'
on runnng:
bash process-youtube-video.sh
while following:
Use a video from YouTube

**Comment by rohit-gupta** [comment]
Oh, seems like you need the vocabulary file. You can create it by using:

```
bash fetch-data.sh
bash preprocess-data.sh CleanOnly 
```

**Comment by vaibh1297** [comment]
Still vocabulary_10.p file is not generated. How to generate it ?

**Comment by rohit-gupta** [comment]
@vaibh1297 Give me a day or so, I will upload the `vocabulary_10.p` file that I generated along with the pre-trained model.

**Comment by vaibh1297** [comment]
ok thanks

**Comment by flexibility2** [comment]
@rohit-gupta ， hi, thanks for your source code, however, I also meet the same problem, could you upload the **vocabulary_10.p** file? Thanks a lot

**Comment by siddish-reddy** [comment]
@rohit-gupta  Hi Rohit
Thanks for providing your source code and also helper scripts.
Can we get vocabulary_10.p file if possible?
Thanks

**Comment by rohit-gupta** [comment]
@siddish-reddy unfortunately I no longer have access to it, as it was in a AWS bucket that has since been deleted. You can still use the code for training, though I recommend against it, as multi-backend Keras (which my code uses) is no longer actively developed.

**Comment by siddish-reddy** [comment]
Yeah, we have tried to train but have got dimension errors in training loop
P.S. Thanks for responding

**Comment by Namzakku** [comment]
Has anyone got the vocabulary_10.p file working? I also got the dimension error..

**Comment by rohit-gupta** [comment]
Hi guys, as I have mentioned previously, I do not recommend using this code for any of your projects.
